tests/Feature/SettingsTest.php

```diff
@@ -51,7 +51,7 @@ public function test_user_can_export_account()
         $response = $this->get(route('settings.sql'));
```

Comment:
With Carbon you can force the date for the tests:
```suggestion
        Carbon::setTestNow(Carbon::create(2021, 11, 1, 7, 0, 0));

        $response = $this->get(route('settings.sql'));
```